internal/module/check.go

```diff
@@ -64,3 +113,27 @@ func getModAndFilter(ctx context.Context, path string, filter func(*semver.Versi
 
 	return checkResults, nil
 }
+
+func getModAndVulnerabilitiesCheck(ctx context.Context, path string, vulnerabilityCheck bool, jsonOutputCheck bool) (map[string]internal.VulnerabilityResult, error) {
+	parser := versionParser{ctx: ctx}
+	vs := make(map[string]internal.VulnerabilityResult)
+	results, err := parser.Parse(path)
+	if err != nil {
+		return nil, err
+	}
+
+	if vulnerabilityCheck {
+		vs = CheckForVulnerabilities(ctx, results)
+		if jsonOutputCheck {
```

Comment:
I have just removed the json output option. 
It would be a good idea to create a new cobra persistentFlag, in order to be available to the command it's assigned to,  as well as every command under that command.